src/components/Methodology.js

```diff
@@ -107,58 +81,41 @@ const Methodology = props => {
+
+          <h4>OSM PBF Extraction</h4>
+          <p> The OSM data used was extracted from a full OSM history PBF file (<a href="https://planet.openstreetmap.org/">Source</a>). The goal of the initial phase of data processing was to have two extracts for each city, one containing 2018 data and the other 2019 data. This was achieved using a tool called <a href="https://osmcode.org/osmium-tool/">Osmium</a>.</p>
+          <p> The first step was to cut the full planet OSM history PBF into a history file for each city using Osmium's 'extract' tool. The tool used geojson boundaries of each city to perform a soft clip for each, generating the new city history PBFs.</p>
+          <p>Next, each of the city history files were time sliced. Using Osmium's 'time-filter' tool each history file was filtered so that the data appeared the same as it would have at a specific point in time in OSM. For each city this was done twice, creating a PBF for April 7th at 10am UTC for both 2018 and 2019. At this point, the new time slice PBFs were in standard OSM PBF format, as opposed to OSM history files.</p>
+          <p>Finally, extraneous Nodes were cleaned from the time slice PBFs using a simple python script. The Nodes in question did not have an spatial information, as they were remnants of features that had been deleted at that point in time.</p>
+
+          <h4>MQM Extent Enhancement</h4>
+          <p>An addition to MQM was incorporated into our methodology this year, to better compare differences over time. The additional feature allows the extent of MQM results to be locked to a specified bounding box, as opposed to being dynamically set by the extent of the input data. This allowed the results of two MQM runs to share the same extent despite small variances in the spatial bound of the input data.</p>
+
+          <h4>Social-Economic Score in MQM</h4>
+          <p> To incorporate the social-economic metrics, we collect data from the U.S. Census Bureau's American Community Survey. This data source includes different demographic information which we used as a proxy for road usage in each city based on each census tract.</p> 
+          <p> With each tract we generated raster layers and grids using city boundaries, we focused on three main data points: population density and car ownership as two different metrics</p>
+          <p> Next, a raster layer is created by each metrics and each city for smaller and unified area of values</p>
+          <p> After that, the zonal statistics tool in QGIS is used to get the mean value of these raster values by MQM grids</p>
+          <p> Finally, once all mean values were calcualted the data was normalized between 0 - 100% scale, we called census_score</p>
+
+          <h4>Re-weight MQM score by usage</h4>
+          <p> Our over-arching question here is if there are multiple area has poor road quality, how can we prioritize it with limited amout of editor efforts</p>
```

Comment:
I like this part, but maybe we can say this at the top of the Methodology page as an introduction of what MQM is.

Reply:
<p> To combine the population and car ownership grids respectively with the MQM map error grids, we assigned a higher weight to the MQM grid because the map errors should still be the dominant factor for determining a city's map data quality. Specifically, we used the following formula: Combined Score = 0.7 × MQM Score + 0.3 × Census Score. For future enhancement, we would like to provide more flexibility for assigning weights to these layers. </p>